src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`
```

Comment:
Nested ternary is not good idea, you can create separate variable for example for partner (husband or wife) and here use like: `${person.partnerName} is my ${partnerStatus}`